### ISSUE TITLE
Room (“Salon”) details 

### ISSUE THREAD

**Issue by giomfo** [dinsic-pim/tchap-ios#42]
Update the room (salon) details screen:

Keep the Riot screen by applying the following changes:
- Display the members like the contacts
- The (+) button is replaced by a button displayed at the top of the list
- Remove access and visibility options from the room settings
- CAUTION: **A public room may become private** (see https://github.com/dinsic-pim/tchap-android/issues/368)
- Remove alias section

design https://zpl.io/aRqKore, https://zpl.io/25ymeZj and https://zpl.io/2vl8y15

**Comment by giomfo** [comment]
This issue blocked by #53

**Comment by giomfo** [comment]
The room settings have been updated.
A new issue have been created to handle the room members list (#66)
